presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -61,13 +61,14 @@ contents:
    access-control.name=file
    security.config-file=etc/rules.json
 
-The config file consists of a list of access control rules in JSON format. The
-rules are matched in the order specified in the file. All
-regular expressions default to ``.*`` if not specified.
+The config file consists of a list of access control rules in JSON format and a
+list of username extraction regex. The rules are matched in the order specified
```

Comment:
Do not inject info about username extraction here. With it, it is not obvious about what regular expressions defaulting to .* next sentence is saying. Actually it is not obvious even without your addition :). 

Maybe make it sth like this will be better:

```
The config file is specified in JSON format. 
* It contains the rules defining which catalog can be accessed by which user (see Catalog Rules below).
* User extraction rules specifying what principals can identify as what users (see Username Extraction below).
```

Then in `Catalog Rules` section add information that rules are traversed from top to bottom. And that catalog/user regex default to .*